test_pair.py

```diff
@@ -0,0 +1,102 @@
+import unittest
+
+import mongomock
+
+import ortho
+import pair as subject
+import session
+
+
+class MyTestCase(unittest.TestCase):
+    def test_it_attempts_b_d_ex_nihilo(self):
+        client = mongomock.MongoClient()
+        collection = client.top
+        s = session.start_session(client)
```

Comment:
If you switch to pytest, you could save yourself some LoC here by moving this into a fixture. Alternatively, unittest supports setup and teardown methods.